qa/sys/centos/bootstrap.sh

```diff
@@ -2,11 +2,3 @@
 
 yum update
 yum install -y java-1.8.0-openjdk-devel.x86_64
-
-##
-# Install logstash manually from a URL
-##
-BRANCH=${LOGSTASH_BRANCH:-'master'}
```

Comment:
My preference would be that test are mostly responsible of testing assertions and make this process as simple as possible, we should avoid any logic in there that is too much tied with the necessary moving parts to run the test (like downloading the artifacts).

for me this point is a blocker to merge this PR, we should get some consensus here 👍 , looking forward to it.